frontend/app/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,91 @@
+"use client";
+
+import React from 'react'
+import { useState } from 'react';
+import { FaBars, FaTimes } from 'react-icons/fa'
+import SearchBar from './SearchBar';
+import Link from 'next/link';
+
+
+
+const NavBar = () => {
+
+  const [nav, setNav] = useState(false)
+
+
+  //   {links.map(({ id, link }) => (
+  //     <li key={id} 
+  //     className='px-4 cursor-pointer capitalize font-medium text-gray-300 hover:scale-105 duration-200'
+  //     >
+  //         {/* {link} */}
+  //         <Link to={link} smooth={500}>{link}</Link>
+  //         </li>
+  // ))}
```

Comment:
Remove dead code -> if you need it later, you can for example store it as a snippet or as a git stash